pkg/scheduler/plugins/proportion/proportion.go

```diff
@@ -87,6 +85,7 @@ func (pp *proportionPlugin) OnSessionOpen(ssn *framework.Session) {
 		guarantee := api.NewResource(queue.Queue.Spec.Guarantee.Resource)
 		pp.totalGuarantee.Add(guarantee)
 	}
+	totalShareResource := pp.totalResource.Clone().Sub(pp.totalGuarantee)
```

Comment:
Should we validate the value of `totalGuarantee` for it may be larger that `totalShareResource`?

Reply:
Just make sure that totalResource = totalGuarantee + totalShareResource, there is no comparison between totalGuarantee and totalShareResource. When totalGuarantee is greater than totalResource, subsequent logic cannot be executed normally. In this case, a panic error will be thrown.

So I don't need to add any extra judgment.